build.sbt

```diff
@@ -222,3 +222,19 @@ lazy val apiModel = (project in file("api-model"))
     libraryDependencies ++= commonDeps ++ httpDeps
   ).dependsOn(modelJVM % "compile->compile;test->test")
   .dependsOn(persistence % "compile->compile")
+
+lazy val publication = (project in file("publication"))
```

Comment:
Should this be aggregated into the `hmda.jar` file or packaged into it's own `.jar`?